diffusionserver.py

```diff
@@ -25,7 +25,7 @@
 class StableDiffusionHandler:
     def __init__(self, token=True):
         self.text2img = StableDiffusionPipeline.from_pretrained(
-        "CompVis/stable-diffusion-v1-4",
+        "runwayml/stable-diffusion-v1-5",
```

Comment:
I forgot to mention that I changed the default model from the 1.4 version of stable diffusion to the new 1.5 version which is a lot better, this also helps a lot with the results quality, on the older model faces and hands were a lot worst and they got better on the new version.